include/openssl/cms.h

```diff
@@ -288,7 +288,7 @@ void *CMS_unsigned_get0_data_by_OBJ(CMS_SignerInfo *si, ASN1_OBJECT *oid,
 int CMS_add1_signing_cert(CMS_SignerInfo *si, ESS_SIGNING_CERT *sc);
 int CMS_add1_signing_cert_v2(CMS_SignerInfo *si, ESS_SIGNING_CERT_V2 *sc);
 
-# ifdef HEADER_X509V3_H
+# ifdef OPENSSL_X509V3_H
```

Comment:
this should be unnecessary, because we include x509v3.h above.

Reply:
See https://github.com/openssl/openssl/pull/9274#discussion_r300690583.